### PR TITLE
Added new docsTaggedWith field/endpoint

### PR DESCRIPTION
**Pull request by sebflipper**
Ship It

### Pre-flight checklist

- [x] Unit tests
- [x] GraphiQL tested
- [x] Client app tested
- [x] Gif added

### CODE REVIEW

**Comment by lpil** [comment]
Wow, that gif is intense

File: lib/fetch.js [hunk 1]
```diff
@@ -100,3 +100,16 @@ export async function fetchNewsArticle(_, args, req) {
 export async function fetchAllNews() {
   return fetchAll('news');
 }
+
+export async function fetchDocsTaggedWith(_, args) {
+  const api = await Prismic.api(PrismicConfig.apiEndpoint);
+  const res = await api.form('everything').ref(api.master()).query(
+    Prismic.Predicates.any('document.tags', [args.tag]))
+    .submit();
```
Comment (lpil) [inline lib/fetch.js]: I feel putting the last `)` on the previous line make it look like `submit` is being called on `query`

**Comment by asavin** [comment]
So as discussed, we should use query params for `tags` on existing queries that return lists of events and news, and more in future.

**Comment by sebflipper** [comment]
@asavin fixed in 996f661 👍

**Comment by asavin** [comment]
Closing this one without merging as it is very stale now. Will re-implement this in a separate branch.

**Pull request closed without merging**